README.md

```diff
@@ -1,2 +1,48 @@
 # Hello-Microverse
 An introduction to the Microverse' gitflow
+
+Additional description about the project and its features.
+
+Built With
+Major languages
+Frameworks
+Technologies used
+Live Demo
+Live Demo Link
+
+Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+👤 Author1
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+👤 Author2
+
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Please work on your README so that it can be easy to read it and understand for a moment you added all required elements you were supposed to add but it is not easy to read them add space where is needed etc. you can refer to this [Template](https://github.com/microverseinc/readme-template)

Reply:
I deleted unnecessary informations from the template of microverse. It is clearer now.